content/docs/components/input/usage.mdx

```diff
@@ -217,3 +217,46 @@ opacity to 1 if you want the placeholder to have a specific color.
   />
 </Stack>
 ```
+
+### Input Methods other than Text
+Date and Time Picker
+
+```jsx
+  <Input
+              placeHolder="Select Date and Time"
+              size="md"
+              backgroundColor="#ffffff"
+              type="datetime-local"
+            />
```

Comment:
I'd go with only this example and add a note about more types and a link to [this list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#input_types)

Also please clean up the indentation

Reply:
### Input Methods other than Text
You can use different types with `Input` such as dateTime, color, search, file etc. 

Date and Time Picker

```jsx
  <Input
     placeHolder="Select Date and Time"
     size="md"
     backgroundColor="#ffffff"
     type="datetime-local"
    />
```
Check for different input types available : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#input_types 

Does this looks good?  Or any other change suggestion?

Reply:
Yes that's exactly what I meant :)